bin/browsertime.js

```diff
@@ -27,6 +27,11 @@ function parseUserScripts(scripts) {
     {});
 }
 
+// 1200 -> 1.2
+function fmt(msec, d=2) {
```

Comment:
Default arguments doesn't work, since we still support node 4.3. Might want to change that soon, but let's do that separately.